doc/ref/configuration/master.rst

```diff
@@ -1321,6 +1322,32 @@ minion IDs for which keys will automatically be rejected. Will override both
 membership in the :conf_master:`autosign_file` and the
 :conf_master:`auto_accept` setting.
 
+.. conf_master:: autosign_grains_dir
+
+``autosign_grains_dir``
+-----------------
```

Comment:
Sorry for the nitpick, but could you line up the dashed line so it's the same length as the line above it? When they are different links it generates a warning when we run Sphinx to build the docs.